### PR TITLE
Add 2 New Array Functions

### PR DESCRIPTION
**Pull request by jokoho48**
add getHightestVaule
add getLowestVaule

### CODE REVIEW

File: addons/arrays/CfgFunctions.hpp [hunk 1]
```diff
@@ -27,6 +27,18 @@ class CfgFunctions
                 description = "A function used to return the element counts in an array. Parameters: Array Example: _types = [0,0,1,1,1,1] call CBA_fnc_getArrayElements Returns: Array element counts (for above example, return would be [0,2,1,4]) Author: Rommel && sbsmac";
                 file = "\x\cba\addons\arrays\fnc_getArrayElements.sqf";
             };
+            // CBA_fnc_getHightestVaule
+            class getHightestVaule
```
Comment (jonpas) [inline addons/arrays/CfgFunctions.hpp]: Should be `getHighestValue`, check spelling (not just here).

File: addons/arrays/CfgFunctions.hpp [hunk 2]
```diff
@@ -27,6 +27,18 @@ class CfgFunctions
                 description = "A function used to return the element counts in an array. Parameters: Array Example: _types = [0,0,1,1,1,1] call CBA_fnc_getArrayElements Returns: Array element counts (for above example, return would be [0,2,1,4]) Author: Rommel && sbsmac";
                 file = "\x\cba\addons\arrays\fnc_getArrayElements.sqf";
             };
+            // CBA_fnc_getHightestVaule
+            class getHightestVaule
+            {
+                description = "Use to find Hightest value with index in a array.";
+                file = "\x\cba\addons\arrays\fnc_getHightestVaule.sqf";
+            };
+            // CBA_fnc_getLowestVaule
```
Comment (jonpas) [inline addons/arrays/CfgFunctions.hpp]: Should be `getLowestValue`, check spelling (not just here).

File: addons/arrays/fnc_getHighestValue.sqf [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+Parameters:
+    _array: Array with Numbers
+
+Example:
+    (begin example)
+    _array = [_array,1] call CBA_fnc_sortNestedArray
+    (end)
+
+Returns:
+    _min: the Max Value
```
Comment (Killswitch00) [inline addons/arrays/fnc_getHighestValue.sqf]: _max: the maximum value

File: addons/arrays/fnc_getLowestValue.sqf [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+Parameters:
+    _array: Array with Numbers
+
+Example:
+    (begin example)
+    _array = [_array,1] call CBA_fnc_sortNestedArray
+    (end)
+
+Returns:
+    _min: the Min Value
```
Comment (Killswitch00) [inline addons/arrays/fnc_getLowestValue.sqf]: _min: the lowest value

File: addons/arrays/fnc_getLowestValue.sqf [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    _index: the Index of the Min Value
```
Comment (Killswitch00) [inline addons/arrays/fnc_getLowestValue.sqf]: ...index of the lowest value

File: addons/arrays/fnc_getHighestValue.sqf [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+    _index: the Index of the Max Value
```
Comment (Killswitch00) [inline addons/arrays/fnc_getHighestValue.sqf]: ...index of the highest value  (or "index of the max value")

**Comment by Killswitch00** [comment]
These functions could have much shorter names:

 * `CBA_fnc_findMin`
*  `CBA_fnc_findMax`

File: addons/arrays/fnc_findMin.sqf [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+Parameters:
+    _array: Array with Numbers
+
+Example:
+    (begin example)
+    _result = [_array] call CBA_fnc_findMin
+    (end)
+
+Returns:
+    _max: smallest value in array
```
Comment (Killswitch00) [inline addons/arrays/fnc_findMin.sqf]: "_min"

**Comment by jokoho48** [comment]
ok done

File: addons/arrays/CfgFunctions.hpp [hunk 8]
```diff
@@ -15,6 +15,18 @@ class CfgFunctions
                 description = "Filter each element of an array via a function.";
                 file = "\x\cba\addons\arrays\fnc_filter.sqf";
             };
+            // CBA_fnc_findMax
+            class findMax
+            {
+                description = "Find smallest numeric value with index in an array.";
```
Comment (Killswitch00) [inline addons/arrays/CfgFunctions.hpp]: This is the description for findMin below.

File: addons/arrays/fnc_findMax.sqf [hunk 9]
```diff
@@ -0,0 +1,41 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_findMax
+
+Description:
+    Find smallest numeric value with index in an array.
```
Comment (Killswitch00) [inline addons/arrays/fnc_findMax.sqf]: findMax looks for the largest ("biggest") value

File: addons/arrays/fnc_findMin.sqf [hunk 10]
```diff
@@ -0,0 +1,41 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_findMin
+
+Description:
+    Find biggest numeric value with index in an array.
```
Comment (Killswitch00) [inline addons/arrays/fnc_findMin.sqf]: findMin looks for the smallest value

**Comment by Killswitch00** [comment]
The merge conflicts in addons/arrays/test.sqf and addons/arrays/CfgFunctions.hpp need to be fixed.

**Comment by jokoho48** [comment]
ok fixed

**Pull request merged**